post/views.py

```
@@ -0,0 +1,89 @@
+        serializer = PostSerializer(post)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    @swagger_auto_schema(
+        operation_id="게시글 삭제",
+        operation_description="게시글을 삭제합니다.",
+        responses={204: "No Content", 404: "Not Found"},
+    )
+    def delete(self, request, post_id):
+        try:
+            post = Post.objects.get(id=post_id)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)